cpp/src/copying/scatter.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <cudf/column/column_view.hpp>
+#include <cudf/table/table_view.hpp>
+#include <cudf/table/table.hpp>
+
+#include <memory>
+
+
+namespace cudf {
+namespace experimental {
+namespace detail {
+
+void scatter(table_view source_table, column_view gather_map,
+	     mutable_table_view destination_table, bool check_bounds = false,
```

Comment:
Can't have in-place `scatter` anymore because of string and non-fixed width element types.